src/app/routes/applications/new-role/new-role.component.ts

```diff
@@ -841,42 +858,12 @@ export class NewRoleComponent implements OnInit, AfterViewInit, OnDestroy {
   }
 
   private async _searchRoleNamespace(keyword: any): Promise<any[]> {
-    if (this.autocompleteTrigger) {
-      this.autocompleteTrigger.closePanel();
-    }
-    
-    this.isAutolistLoading = true;
     let retVal = [];
-
-    if (keyword) {
       let word = undefined;
-      if (!keyword.trim && keyword.name) {
-        word = keyword.name;
-      } else {
-        word = keyword.trim();
-      }
-
-      if (word.length > 2) {
-        word = word.toLowerCase();
-        try {
           retVal = await this.iamService.iam.getENSTypesBySearchPhrase({
```

Comment:
Where are we catching if this request fails?

Reply:
reverted it